src/main/java/uk/ac/ebi/eva/contigalias/dus/NCBIAssemblyReportReader.java

```diff
@@ -44,7 +44,8 @@ protected void parseReport() throws IOException, NullPointerException {
                 parseAssemblyData(line);
             } else if (!line.startsWith("#")) {
                 String[] columns = line.split("\t", -1);
-                if (columns.length >= 6 && columns[5].equals("=")) {
+                if ((columns.length >= 6 && columns[5].equals("=")) ||
```

Comment:
In line 102: `chromosomeEntity.setRefseq(columns[6]);` needs to be checked for `na`

Reply:
updated

Reply:
done